build/root/usr/local/bin/entitle.sh

```diff
@@ -65,10 +66,10 @@ if [ ! -e "$ENTITLEMENT_PEM" ]; then
     exit 1
 fi
 
-echo "INFO: Deploying the entitlement with PEM key from ${ENTITLEMENT_PEM}"
-toolbox/entitlement/deploy.sh --pem "${ENTITLEMENT_PEM}"
+echo "INFO: Deploying the entitlement with PEM key from ${ENTITLED_MIRROR_CLIENT_CREDS}"
+toolbox/entitlement/deploy.sh --yum-client-auth="${ENTITLED_MIRROR_CLIENT_CREDS}"
 
-if ! toolbox/entitlement/wait.sh; then
-    echo "FATAL: Failed to properly entitle the cluster, cannot continue."
-    exit 1
-fi
```

Comment:
can you get this test to work? it may not be straightforward, but it would allow catching mirror errors earlier than in the driver container not deploying ...
or add a GPU Operator test to detect when the driver fails to build

Reply:
Good point